### ISSUE TITLE
Full UI: QuestionFragment [Blocked: #15, #28, #110, #111, #158]

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#27]
Content cards can be paired with questions that hold an interaction for the purpose of quizzing the learner on a particular concept. Questions are used when reviewing concepts, training skills, or within an exploration as part of a story. See 3.2.4 of the technical architecture document for the interactions that will be supported in the prototype.

**Comment by BenHenning** [comment]
This is blocked by #15 and interactions architecture work: #28. Other interactions do not need to be completed for this work to begin.

Note that this should be architected such that it can be easily reused in both practice session and exploration contexts.

**Comment by BenHenning** [comment]
Corresponds to the [training tab](https://xd.adobe.com/view/bee1fced-33bd-4f69-6a89-2c30fe1c4f65-c8d0/screen/3088dfe8-937a-4fa4-8260-4ee531503844/Home-Page-1-Start-Learning-Pre-Scroll-17) and [question page](https://xd.adobe.com/view/bee1fced-33bd-4f69-6a89-2c30fe1c4f65-c8d0/screen/8fbb663a-39aa-4be1-83bd-73bde6f85dcc/Lesson-1-a-i-Exploration-Player-Fraction-Input-No-). We should figure out how individual questions are represented, how the entire training session works, and what the fragment of the top-level training selection should look like.

**Comment by BenHenning** [comment]
This is also blocked by #110, #111, and #158.

**Comment by BenHenning** [comment]
This is a duplicate of #40.
